src/logic/safe/store/models/types/gateway.d.ts

```diff
@@ -16,6 +16,21 @@ import {
   TransactionStatus,
 } from '@gnosis.pm/safe-react-gateway-sdk'
 
+/**
+ * We can't use the enum values from the SDK directly when comparing to strings
+ * Not sure if a bug or a feature 🤷
+ */
+export const LocalTransactionStatus: Record<string, TransactionStatus> = {
+  AWAITING_CONFIRMATIONS: 'AWAITING_CONFIRMATIONS',
+  AWAITING_EXECUTION: 'AWAITING_EXECUTION',
+  CANCELLED: 'CANCELLED',
+  FAILED: 'FAILED',
+  SUCCESS: 'SUCCESS',
+  PENDING: 'PENDING',
```

Comment:
With the intended separation of backend-returned transaction details from their client-side status counterparts, I'm not too sure about keep these in the same enum. (I have prepared a PR on the SDK to remove them as we discussed: https://github.com/gnosis/safe-react-gateway-sdk/pull/40)

Reply:
That would be problematic atm, because we're using TransactionStatus as a type for both local and CGW statuses. I suggest we merge your SDK changes and adapt the safe-react code in a separate PR.

Reply:
We could just move to a union type and it shouldn't be an issue, e.g. `Record<string, TransactionStatus | LocalTransactionStatus>`

Reply:
It would mean replacing TransactionStatus everywhere with a local type. And also dealing with the type mismatch between the txStatus from CGW (which will be TransactionStatus sans PENDING) and the local one. I'd rather not, not in this PR.